float64 probability
int64 x_center
int64 y_center
int16 id
string Class
